server/svix-server/src/core/types.rs

```diff
@@ -427,6 +428,35 @@ impl Validate for EventTypeNameSet {
     }
 }
 
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct RetrySchedule(pub Vec<Duration>);
+json_wrapper!(RetrySchedule);
+
+impl Serialize for RetrySchedule {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: serde::Serializer,
+    {
+        let mut seq = serializer.serialize_seq(Some(self.0.len()))?;
+        for e in self.clone().0 {
+            seq.serialize_element(&e.as_secs())?;
+        }
+        seq.end()
```

Comment:
This is fine, but you can literally do the reverse of what we did in deserialize (simpler):
```rust
let buf: Vec<u64> = self.0.map(|x| x.as_secs()).collect();
buf.serialize(serializer)
```